### PR TITLE
New Onfido SDK watermark design

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
Onfido has recently introduced a new branding design. The SDKs' footer/watermark image needs to be updated. 
Document selection screen with new design
Selfie capture (fullscreen) screen with new design

# Solution
Replace current footer/watermark image with new design assets provided.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/assets/watermark_night.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....57-.8l.03.74h1.18v-7.1h-1.22zm-1.3 6.12c-.73 0-1.2-.56-1.2-1.44 0-.9.5-1.5 1.2-1.5.84 0 1.28.7 1.28 1.33v.3c0 .6-.43 1.32-1.3 1.32zm8.187-1.44c0-1.53-.84-2.53-2.3-2.53-1.4 0-2.42 1.03-2.42 2.5 0 1.45.9 2.5 2.4 2.5 1.43 0 2.1-.8 2.28-1.65h-1.15c-.08.3-.35.68-1.08.68-.7 0-1.17-.45-1.26-1.25h3.5v-.25zm-2.28-1.6c.6 0 1.02.34 1.12 1.1h-2.33c.13-.72.6-1.1 1.2-1.1zm6.01-.94c-.74 0-1.25.3-1.54.76l-.04-.7h-1.17V19h1.22v-2.78c0-.67.43-1.12 1.1-1.12.62 0 .94.4.94 1.1V19h1.23v-3.08c0-1.13-.64-1.87-1.73-1.87zm5.604 3.25c0 .54-.23.76-.63.76-.4 0-.6-.24-.6-.76v-2.22h1.73v-.98h-1.73v-1.45h-1.15v1.45h-.8v1h.73v2.18c0 1.16.68 1.77 1.78 1.77 1.14 0 1.76-.64 1.76-1.74h-1.08zm3.25-4.8c0-.44-.35-.77-.8-.77s-.8.33-.8.77c0 .46.35.78.8.78s.8-.32.8-.78zm-1.42 6.5h1.22v-4.9h-1.22V19zm5.2-1.7c0 .54-.23.76-.63.76-.4 0-.6-.24-.6-.76v-2.22h1.73v-.98h-1.73v-1.45h-1.15v1.45h-.8v1h.73v2.18c0 1.16.68 1.77 1.78 1.77 1.14 0 1.76-.64 1.76-1.74h-1.08zm4.872-3.2l-1.05 3.2a3.46 3.46 0 0 0-.13.55h-.03l-.14-.55-1.04-3.2h-1.33l1.9 5.02-.08.2c-.2.57-.56.7-1 .7a1.16 1.16 0 0 1-.47-.1l-.08 1.03c.18.06.5.1.8.1 1 0 1.5-.47 1.97-1.7l.14-.37 1.8-4.9h-1.24z" fill="#fff"/><path stroke="#fff" stroke-width=".7" d="M75.35 6v19"/></svg>
```
Comment (stefaniacardenas) [inline src/components/Theme/assets/watermark_night.svg:1]: Are these svg optimised, yeah?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1034-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/assets/watermark_night.svg (hunk 1 above)
Comment (kopijunkie) [reply src/components/Theme/assets/watermark_night.svg:1]: Yeap. These have been optimised.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1034-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1034-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**